dnf/util.py

```diff
@@ -644,29 +644,51 @@ def _is_file_pattern_present(specs):
 
 
 def is_container():
+    """Returns true if DNF is running in a container,
+       false otherwise.
+    """
+    # This attempts to reimplement
+    # https://github.com/ostreedev/ostree-rs-ext/blob/b0d1c1e4748c8708149bb6c92998da623260c4c6/lib/src/container_utils.rs#L20
+    # in python.
+    container_env_var = 'container'
+    container_env_file = '/run/.containerenv'
+    docker_env_file =  '/.dockerenv'
+    # https://stackoverflow.com/questions/20010199/how-to-determine-if-a-process-runs-inside-lxc-docker
+    if os.getenv(container_env_var) is not None or os.path.isfile(docker_env_file) or os.path.isfile(container_env_file):
+        return True
+    else:
+        return False
```

Comment:
Just `return os.getenv(container_env_var) is not None or os.path.isfile(docker_env_file) or os.path.isfile(container_env_file)` right?